exporter/src/main.rs

```diff
@@ -168,23 +178,28 @@ fn take_screenshot(
         if let Some(progress) = &progress {
             progress.inc(1);
         }
+
+        if is_root_movie_clip_at_end(&player) {
+            break;
+        }
```

Comment:
That will break some animations, it has to be another CLI option

Reply:
Made a new cli command and raised under #20929